pkg/test/ginkgo/cmd_runsuite.go

```diff
@@ -160,7 +160,32 @@ func (opt *Options) Run(suite *TestSuite, junitSuiteName string) error {
 
 	tests, err := testsForSuite()
 	if err != nil {
-		return err
+		return fmt.Errorf("failed reading origin test suites: %w", err)
+	}
+
+	if len(os.Getenv("OPENSHIFT_SKIP_EXTERNAL_TESTS")) == 0 {
+		fmt.Fprintf(opt.Out, "Attempting to pull tests from external binary...\n")
```

Comment:
When I run the CSI test suite, this message gets printed correctly. But when I run a single test from the CSI test suite (i.e., `run-test` option) I don't see this message at all.